wsmaster/che-core-api-devfile/src/test/java/org/eclipse/che/api/devfile/server/DevfileManagerTest.java

```diff
@@ -79,8 +78,8 @@ public void testValidateAndConvert() throws Exception {
   }
 
   @Test(
-      expectedExceptions = JsonProcessingException.class,
-      expectedExceptionsMessageRegExp = "Unrecognized field \"foos\" [\\w\\W]+")
+      expectedExceptions = DevfileFormatException.class,
```

Comment:
You changed test class without changing tested class source code. How is it supposed to work? Maybe I missed something.

Reply:
It is due to adding `"additionalProperties": false`  in schema. So now the test raises exception during validation (but this doesn't matter for manager itself).